packages/ketcher-core/src/application/formatters/identifyStructFormat.ts

```diff
@@ -26,7 +27,11 @@ export function identifyStructFormat(
     if (JSON.parse(sanitizedString)) {
       return SupportedFormat.ket;
     }
-  } catch (er) {} // eslint-disable-line
+  } catch (er) {
+    KetcherLogger.showExceptionLocation(
```

Comment:
General comment for everywhere – to pass exception to logger. If someone has any error, then you'll be able to find the place, where error has occurred, but won't be able to understand the error it self.

Reply:
Done